app/js/components/demographics.jsx

```diff
@@ -121,147 +106,124 @@ export default class Demographics extends React.Component {
-   */
     return (
-        <div>
-            <form className="form-horizontal" onSubmit={this.handleSave}>
-                <div className="form-group">
-                        <label className="control-label col-sm-2">Gender</label>
-                        <div className="col-sm-5">
-                            <label className="radio-inline">
-                                <input type="radio" id="inlineRadio1" value="option1"name="male" type="radio" value="M" checked={this.state.gender === "M"} onChange={this.setGender}/> Male
+      <div>
+        <form className="form-horizontal" onSubmit={this.handleSave}>
+          <div className="form-group">
+            <label className="control-label col-sm-2">Gender</label>
+            <div className="col-sm-5">
+              <label className="radio-inline">
+                <input type="radio" id="inlineRadio1" value="option1" name="male" type="radio" value="M" checked={this.state.gender === "M"} onChange={this.setGender} /> Male
                             </label>
-                            <label className="radio-inline">
-                                <input type="radio" id="inlineRadio2" value="option2" name="female" type="radio" value="F" checked={this.state.gender === "F"} onChange={this.setGender}/> Female
+              <label className="radio-inline">
+                <input type="radio" id="inlineRadio2" value="option2" name="female" type="radio" value="F" checked={this.state.gender === "F"} onChange={this.setGender} /> Female
                             </label>
-                        </div>
-                </div>
-
-                <div className="form-group ">
-                    <label className="control-label col-sm-2">Birth Date</label>
-                    <div className="col-sm-5">
-                        <DatePicker
-                            dateFormat="DD-MM-YYYY"
-                            className="form-control"
-                            name="birthdate"
-                            value={this.state.birthdate} onChange={this.handleTimeChange('birthdate')}/>
-                    </div>
-                    <label className="control-label col-sm-2"> Estimated </label>
-                    <div className="col-sm-2">
-                        <input
-                            name="estimates"
-                            type="checkbox"
-                            defaultChecked={this.state.birthDateEstimated}
-                            onChange={this.setBirthDateEstimate}/>
-                    </div>
+            </div>
+          </div>
+
+          <div className="form-group ">
+            <label className="control-label col-sm-2">Birth Date</label>
+            <div className="col-sm-5">
+              <DatePicker
+                dateFormat="DD-MM-YYYY"
+                className="form-control"
+                name="birthdate"
+                value={this.state.birthdate} onChange={this.handleTimeChange('birthdate')} />
+            </div>
```

Comment:
The onchange property could go on a new line to be consistent with the previous ones

Reply:
@jacksono  well noted. Working on above.